src/coreComponents/constitutive/permeability/ParallelPlatesPermeability.hpp

```diff
@@ -33,26 +33,31 @@ class ParallelPlatesPermeabilityUpdate : public PermeabilityBaseUpdate
 
   ParallelPlatesPermeabilityUpdate( arrayView3d< real64 > const & permeability,
                                     arrayView3d< real64 > const & dPerm_dPressure,
-                                    arrayView4d< real64 > const & dPerm_dDispJump )
+                                    arrayView4d< real64 > const & dPerm_dDispJump,
+                                    bool const updateTransversalComponent )
     : PermeabilityBaseUpdate( permeability, dPerm_dPressure ),
-    m_dPerm_dDispJump( dPerm_dDispJump )
-  {}
+    m_dPerm_dDispJump( dPerm_dDispJump ),
+    m_numDimensionsToUpdate( 3 )
+  {
+    m_numDimensionsToUpdate = updateTransversalComponent ? 3 : 2;
+  }
 
   GEOS_HOST_DEVICE
   void compute( real64 const & oldHydraulicAperture,
                 real64 const & newHydraulicAperture,
+                real64 const & dHydraulicAperture_dNormalJump,
                 arraySlice1d< real64 > const & permeability,
                 arraySlice2d< real64 > const & dPerm_dDispJump ) const
   {
     GEOS_UNUSED_VAR( oldHydraulicAperture );
 
     real64 const perm  = newHydraulicAperture*newHydraulicAperture / 12.0;
-    real64 const dPerm = newHydraulicAperture / 6.0;
+    real64 const dPerm_dHydraulicAperture = newHydraulicAperture / 6.0;
 
-    for( int dim=0; dim < 3; dim++ )
+    for( int dim=0; dim < m_numDimensionsToUpdate; dim++ )
     {
-      permeability[dim]     = perm;
-      dPerm_dDispJump[dim][0]  = dPerm;
+      permeability[dim]        = perm;
+      dPerm_dDispJump[dim][0]  = dPerm_dHydraulicAperture * dHydraulicAperture_dNormalJump;
```

Comment:
so this one is more like a correction, previously there was a derivative but it was not accounting the mapping?